api/Console.json

```diff
@@ -1046,22 +1070,22 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/profile",
           "support": {
             "chrome": {
-              "version_added": "53"
+              "version_added": "4"
             },
             "chrome_android": {
-              "version_added": "53"
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "16"
             },
             "firefox_android": {
-              "version_added": "10"
+              "version_added": "16"
```

Comment:
Since this is increasing the version from 10 to 16, would you mind checking where the original data came from, to see if there might just be something surprising going on here? The same request goes for a few following changes to firefox_android data.

Reply:
The original data came from the initial MDN wiki migration, so I'm betting that the data was incorrect.